packages/core/src/project/Version.ts

```diff
@@ -0,0 +1,122 @@
+import semver = require("semver");
+
+/**Helper functions for updating the project config */
+
+export default class Version {
```

Comment:
I think this should be in Package as opposed to Project.

Reply:
Probably tie in with SFPPackage

Reply:
Should have a constructor which takes in a package and  operate on top of it

Reply:
Yeah will do constructor with package input and exec can be to call the update method. But I will still leave the getter methods as public.

Reply:
Changes made - I think it makes it a little unclear, but changes made.